docs/simulators/DualSPHysics.md

```diff
@@ -48,6 +52,395 @@ task.download_outputs()
+Input archive size: 1.09 MB
+Uploading input archive...
+100%|██████████████████████████████████████████████████████████████████████████████| 1.09M/1.09M [00:01<00:00, 715kB/s]
+Local input directory successfully uploaded.
+
+■ Task u8v7p1v7wfyvvkyc0iq0s632k submitted to the queue of the Machine Group api-57yp64vdyz50a8fx0ttuz2n90 with n2d-highcpu-64 machines.
+Number of tasks ahead in the queue: 0
+Simulation metadata logged to: inductiva_output/task_metadata.json
+Task u8v7p1v7wfyvvkyc0iq0s632k configurations metadata saved to the tasks metadata file task_metadata.json in the current working directory.
+· Consider tracking the status of the task via CLI:
+	inductiva tasks list --id u8v7p1v7wfyvvkyc0iq0s632k
+· Or, tracking the logs of the task via CLI:
+	inductiva logs u8v7p1v7wfyvvkyc0iq0s632k
+· You can also get more information about the task via the CLI command:
+	inductiva tasks info u8v7p1v7wfyvvkyc0iq0s632k
+
+■ Task u8v7p1v7wfyvvkyc0iq0s632k successfully queued and waiting to be picked-up for execution...
+The task u8v7p1v7wfyvvkyc0iq0s632k is about to start.
+■ Task u8v7p1v7wfyvvkyc0iq0s632k has started and is now running remotely.
+■ Task u8v7p1v7wfyvvkyc0iq0s632k completed successfully.
+Downloading stdout and stderr files to u8v7p1v7wfyvvkyc0iq0s632k...
+Partial download completed to u8v7p1v7wfyvvkyc0iq0s632k.
+Successfully requested termination of MachineGroup(name="api-57yp64vdyz50a8fx0ttuz2n90").
+Termination of the machine group freed the following quotas:
+
+                                               FREED BY RESOURCE     CURRENT USAGE     MAX ALLOWED
+ Maximum number of VCPUs                       64                    0                 1000
+ Maximum simultaneous instances                1                     0                 100
+ Maximum price per hour across all instances   0.6727                0                 270
+
+
+Task status: success
+Wall clock time:  0:09:35
+Time breakdown:
+	Input upload:              1.81 s
+	Time in queue:             10.71 s
+	Container image download:  1.23 s
+	Input download:            0.09 s
+	Input decompression:       0.01 s
+	Computation:               0:06:20
+	Output upload:             0:03:01
+Data:
+	Size of zipped output:    3.52 GB
+	Size of unzipped output:  5.35 GB
+	Number of output files:   2541
+
+```
+
+That's it!
+
+We can now donwload the results to our local machine using Inductiva's CLI:
+```bash
+inductiva tasks download u8v7p1v7wfyvvkyc0iq0s632k
+```
+Downloading and decompressing data will take a few minutes (depending on your
+network connection):
+```bash
+Downloading simulation outputs to inductiva_output/u8v7p1v7wfyvvkyc0iq0s632k/output.zip...
+100%|█████████████████████████████████████████████████████████████████████████████| 3.52G/3.52G [04:43<00:00, 12.4MB/s]
+Uncompressing the outputs to u8v7p1v7wfyvvkyc0iq0s632k...
+```
+
+As usual, the results are placed inside the `inductiva_output` folder, and then
```

Comment:
As usual, the results are placed in the inductiva_output folder, within a subfolder named after the task. Earlier, we set a variable for the internal directory where all outputs would be placed (`dirout`), which was instantiated as CaseTurbine_out. Let’s check its contents: